cron/data/projects.release.csv

```diff
@@ -1,4 +1,9 @@
 repo,metadata
+#Basic
```

Comment:
What do basic and 300 redirects mean?

Reply:
Basic, is just regular repos which run without any special corner case. The 300 redirect one is explained in this PR: https://github.com/ossf/scorecard/pull/670